Development-Guide/Vim-Astyle-Guide/README.adoc

```diff
@@ -6,6 +6,8 @@ In neovim there are commands to format your file into astyle.
 == Installing Autoformat to use with Astyle
 First install Astyle following the guide outlined in the Style Guide.
 
+Be sure to create the astylerc and know the file path to it.
```

Comment:
We can remove this because we will explain in detail where astylerc is later in the guide